oauth2_http/java/com/google/auth/oauth2/JwtClaims.java

```diff
@@ -61,8 +63,15 @@
   @Nullable
   abstract String getSubject();
 
+  /**
+   * Returns additional claims for this object; <b>do not mutate</b> the returned map.
```

Comment:
Why not? Should this simply return an ImmutableMap instead?

Reply:
We generally don't want to expose Guava types in the API surface, so we make it a generic map and document it as immutable

Reply:
Yes, this is an ugliness in the Java Collections API. I tend to prefer simply returning a mutable copy to avoid UnsupportedOperationExceptions, but if that's not a good idea maybe change "; <b>do not mutate</b> the returned map" to "as an immutable map" or perhaps write as, "The map is not guaranteed to be mutable." The way it's written now sounds like you could mutate it but bad things would then happen.

Reply:
updated